doc/source/api.rst

```diff
@@ -1262,7 +1262,7 @@ Modifying and Computations
    Index.argmax
    Index.copy
    Index.delete
-   Index.diff
+   Index.difference
```

Comment:
But I think this does not belong in this section at all, as this is not a 'computation' function

I would rather put it together with the other set operation methods